src/pages/game/game-page.jsx

```diff
@@ -20,17 +20,20 @@ export default function GamePage({ match }) {
     setGameId(match.params.gameId);
   }, [match]);
 
-  function connectWebSocket(pid, gid) {
+  function vote(points) {
+    socket.vote(points);
+  }
+
+  function addPlayer(pid, gid, cb) {
     socket.register(gid, pid, data => {
       console.log('Got a message from the server', data);
+      setPlayerId(pid);
+      setGameId(data.game.id);
       setPlayers(data.game.players);
+      cb(data);
```

Comment:
We don't want to call this function more than once, because the component who owns it won't exist after the first time